packages/web-react/src/components/Tabs/README.md

```diff
@@ -62,34 +62,45 @@ const selectTab = useCallback((id) => {
 </UncontrolledTabs>
 ```
 
+## Custom Spacing
+
+You can use the `spacing` prop to apply custom spacing between items. The prop
+accepts either a spacing token (e.g. `space-100`) or an object with breakpoint keys and spacing token values.
+
+Default spacing:
+
+```jsx
+<Tabs spacing="space-1200">{/* Tabs content */}</Tabs>
+```
+
+Custom responsive spacing:
+
+```jsx
+<Tabs spacing={{ mobile: 'space-400', tablet: 'space-800' }}>{/* Tabs content */}</Tabs>
+```
+
 ### Tabs
 
 #### API
 
-| Name                | Type                   | Default | Required | Description                                  |
-| ------------------- | ---------------------- | ------- | -------- | -------------------------------------------- |
-| `selectedTab`       | [`string` \| `number`] | —       | ✓        | Identification of the selected tab           |
-| `toogle`            | `Function`             | —       | ✓        | Toggle function which accept tab ID as input |
-| `children`          | `any`                  | —       | ✕        | Child component                              |
-| `onSelectionChange` | `(id: TabId) => void`  | —       | ✕        | When the state of the selected panel changes |
-
-On top of the API options, the components accept [additional attributes][readme-additional-attributes].
-If you need more control over the styling of a component, you can use [style props][readme-style-props]
-and [escape hatches][readme-escape-hatches].
+| Name                | Type                                                             | Default | Required | Description                                           |
+| ------------------- | ---------------------------------------------------------------- | ------- | -------- | ----------------------------------------------------- |
+| `selectedTab`       | [`string` \| `number`]                                           | —       | ✓        | Identification of the selected tab                    |
+| `toogle`            | `Function`                                                       | —       | ✓        | Toggle function which accept tab ID as input          |
+| `children`          | `any`                                                            | —       | ✕        | Child component                                       |
+| `onSelectionChange` | `(id: TabId) => void`                                            | —       | ✕        | When the state of the selected panel changes          |
+| `spacing`           | [`SpaceToken` \| `Partial<Record<BreakpointToken, SpaceToken>>`] | —       | ✕        | Apply [custom spacing](#custom-spacing) between items |
```

Comment:
Hmm, I got your point but I don't have a better idea at the moment.

I would discuss this topic with https://jira.almacareer.tech/browse/DS-1424.

Reply:
I don't think we will have vertical spacing here as well. And If we will, we will have to deprecate this `spacing`. 

And it probably won't be in Figma, because I think there is an auto layout, so it is more flexible.